src/services/token.service.ts

```diff
@@ -55,6 +67,23 @@ export class TokenService implements TokenServiceAbstraction {
         return this.refreshToken;
     }
 
+    async toggleTokens(): Promise<any> {
+        const token = await this.getToken();
+        const refreshToken = await this.getRefreshToken();
+        const timeout = await this.storageService.get(ConstantsService.vaultTimeoutKey);
+        const action = await this.storageService.get(ConstantsService.vaultTimeoutActionKey);
+        if ((timeout != null || timeout === 0) && action === 'logOut') {
```

Comment:
@kspearrin I was trying to figure out why the check here also includes an explicit comparison for 0 while the check in `skipTokenStorage` is only for `!= null` : this is following the same pattern as `cryptoStorage`.

Reply:
Yea, I am not sure... I guess it doesn't hurt to leave it here...